integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/ChromeWebDriverProvider.java

```diff
@@ -86,12 +87,17 @@ public WebDriver createWebDriver() {
     System.setProperty(
         "webdriver.chrome.driver", ITUtils.joinPath(downLoadDir, chromeDriverBinName));
     ChromeOptions chromeOptions = new ChromeOptions();
+
+    // Display the web interface during testing
+    if (Strings.isEmpty(System.getenv("DISPLAY_WEBPAGE_IN_TESTING"))) {
```

Comment:
I think default is not show WEB page,
Only front-end developer set `DISPLAY_WEBPAGE_IN_TESTING=true`, then display the web interface during testing.

so condition need change to 
```
if (!Strings.isEmpty(System.getenv("DISPLAY_WEBPAGE_IN_TESTING"))) {
```

Reply:
Hi @xunliu, I think adding `--headless` will not show the browser window. By default, we won't have `DISPLAY_WEBPAGE_IN_TESTING` configured. So the `isEmpty` will result in true, meaning a headless mode.

Reply:
Yes, default is `--headless` mode will not show the browser, because a lot of contribute is backend engineer, them not focus web front-end.

Reply:
@zivali You are right!